lib/cli/init.ts

```diff
@@ -54,3 +62,42 @@ export function init(): void {
     format,
   });
 }
+/**
+ * Initializes UI configuration file
+ * @param {opts}
+ */
+export function initConfig(opts: {
+  cwd: string;
+  directories?: any[];
+  configuration?: string | undefined;
+  configFilePath?: string | undefined;
+}) {
+  let baseConfig = {};
+  const configFilePath =
+    opts?.configFilePath || path.join(opts.cwd, 'config.json');
+  const configFolderPath = path.join(opts.cwd, 'config');
+  const envFilePath = path.join(opts.cwd, '.env');
+  // checking the existances of the legacy config file
+  if (fs.existsSync(configFilePath) && fs.lstatSync(configFilePath).isFile()) {
+    baseConfig = configLoaderSync({
+      cwd: opts.cwd,
+      directories: opts.directories,
+      configFilePath: opts.configFilePath,
+    });
+  } else if (
+    fs.existsSync(configFolderPath) &&
+    fs.lstatSync(configFolderPath).isDirectory()
+  ) {
+    if (!_.isNil(opts.configuration)) {
+      process.env.NODE_CONFIG_ENV = opts.configuration;
+    }
+    if (fs.existsSync(envFilePath)) {
+      require('dotenv').config({
```

Comment:
The coverage still says that this line is not getting hit. 
https://codecov.io/gh/LabShare/lsc/pull/163/diff?src=pr&el=tree#D10-95

Does it actually load set environment variables from .env file in tests?

Reply:
ahh will add that test as well